en/flying/pre_flight_checks.md

```diff
@@ -0,0 +1,73 @@
+# Preflight Sensor/Estimator Checks
```

Comment:
This is essentially the same as merged to DevGuide in https://github.com/PX4/Devguide/pull/510#issuecomment-384519648 but I have changed the introduction to explain were you might see the messages and link to EKF info - as end users might not know that level of detail about our estimators.